src/components/App/App.jsx

```diff
@@ -26,7 +27,7 @@ export default function App() {
       <Routes>
         <Route path="/" element={<Home encounters={encounters} />} />
         <Route path="/login" element={<Login />} />
-        {/* <Route path="/:id" element={<EncounterDetails />}/> */}
+        <Route path="/details" element={<EncounterDetails encounters={encounters}/>}/>
```

Comment:
Are you thinking that we'll hook this back to the variable in a subsequent PR when we hook the encounters to their previews? I think that's totally fine, just want to confirm that it's intentional.

Reply:
Correct, I was using that url for the interim as a placeholder. The updated one that I noted in the other comment thread above as well will come from the `useParams` hook.